contrib/ohi-release-notes/run.sh

```diff
@@ -65,7 +68,7 @@ EOM
 # parsing flags
 EXCLUDED_DIRECTORIES=".github"
 IS_HELD_FAIL="--fail"
-DICTIONARY=".github/rt-dictionary.yaml"
+DICTIONARY=""
```

Comment:
If the new default is going to be empty, the test below ([line 118, sorry, I cannot leave this comment there](https://github.com/newrelic/release-toolkit/pull/152/files#diff-611cd2df9226104e98791e3c12c2f96eb8e741fbf973033eb4b7d4c6abf8e019R118-R122)) instead of testing if the file exists should test that the string is empty.

If it is not empty, it should use the flag `--dictionary` and so `rt` fail explicitly if the file does not exist.

Reply:
that test is done before, and if empty it gets the default dictionary, so it could never be an empty string.

Reply:
In that case, `$DICTIONARY` is not going to be empty. So the test in line [118](https://github.com/newrelic/release-toolkit/pull/152/files#diff-611cd2df9226104e98791e3c12c2f96eb8e741fbf973033eb4b7d4c6abf8e019R118-R122) does not make sense anymore, right?

Reply:
it has the same sense as it was before, to check if the specified file exists before passing it to rt

Reply:
oh now it make sesnse againt to have that default so is aligned with the action. i will change it

Reply:
done the defaults now match , between action and script